pkg/gitconfig/config.go

```diff
@@ -383,7 +383,7 @@ func ParseConfig(r io.Reader) *Config {
 	c.raw.WriteString(strings.Join(lines, "\n"))
 	c.raw.WriteString("\n")
 
-	debug.Log("processed config: %s\nvars: %+v", c.raw.String(), c.vars)
+	// debug.Log("processed config: %s\nvars: %+v", c.raw.String(), c.vars)
```

Comment:
no debug?

Reply:
We're way to noisy already. Until we have better log filtering I want to disable some of them.